### ISSUE TITLE
manage.py sync_customers broken - AttributeError: invoices

### ISSUE THREAD

**Issue by mcastle** [pinax/pinax-stripe#582]
Running manage.py sync_customers is broken and throws an AttributeError: invoices.

```
File "/pinax/stripe/actions/invoices.py", line 134, in sync_invoices_for_customer
    for invoice in customer.stripe_customer.invoices().data:
  File "/stripe/stripe_object.py", line 85, in __getattr__
    raise AttributeError(*err.args)
AttributeError: invoices
```
I first noticed this after upgrading to pinax-stripe 4.4.0 and stripe-python 2.4.0.

**Comment by bonidjukic** [comment]
@mcastle If you're insterested this is a quick hack we came up with to be able to support latest `pinax-stripe` and `stripe-python` versions: https://github.com/bonidjukic/pinax-stripe/commit/80e618eed0eebf4f8072ec87b8e7db99b57008b3

Please keep in mind that we only tested the above mentioned `sync_customers`, we still have to test if this works generally.

BTW, this is the `stripe-paython` "deprecation" commit which seems to have introduced regressions in `pinax-stripe`: https://github.com/stripe/stripe-python/commit/d416e9e68c8eba804535541ce29d2d8f2d22b08c

**Comment by blueyed** [comment]
@bonidjukic 
Please consider creating a PR, then it should also be visible if this code is covered in tests (probably not).

**Comment by streeter** [comment]
The problem is that the tests [mock the call out to `invoices.sync_invoices_for_customer`](https://github.com/pinax/pinax-stripe/blob/master/pinax/stripe/tests/test_commands.py#L79). Or, in the tests for the function itself, that is [also mocked](https://github.com/pinax/pinax-stripe/blob/master/pinax/stripe/tests/test_actions.py#L1671-L1674). The mocks aren't mocking a specific version of the stripe-python library, they are mocking whatever you want them to.

Basically, a fix would involve replicating the code to fetch all the invoices for the given user, doing the same thing the removed code did: https://github.com/stripe/stripe-python/commit/d416e9e68c8eba804535541ce29d2d8f2d22b08c#diff-0399a8a9696e3c2e5288d50b66a3c81cL30

```python
def get_invoices_for_customer(customer):
	params = {customer_id: customer.id}
	invoices = Invoice.list(customer.api_key, **params)
	return invoices
```
